services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
```

Comment:
The parameter `movie_show_time` is expected to be a `str`, but it should be a `datetime` object to match the `DateTimeField` in the model. Consider changing the parameter type to `datetime`.